### DISCUSSION TITLE
How do I get the sidebar to minimize

### DISCUSSION THREAD

**Discussion by GenericUserIsTaken** [artsyfriedchicken/EdgyArc-fr#11] [General]
My sidebar doesn't minimize. I'm on windows, firefox v123.0.1. This might be an issue but its probably a setting I didn't apply or something so I though I would ask here. This might be caused by me deleting translucent-arc.css and translucent-base.css due to the transparency setting being broken.

**Comment by artsyfriedchicken** [comment]
please use `uc.tweak.af.not-translucent` to turn off transparency. dont just delete css files lmao
